pkg/perf/perf.go

```diff
@@ -116,7 +116,7 @@ func MapFromDump(logger log.Logger, fs fs.FS, fileName string) (Map, error) {
 		}
 		// Some runtimes update their dump all the time (e.g. libperf_jvmti.so),
```

Comment:
Yeah this will be the norm in case you attach to a running process like parca will do.